cmd/admission-webhook-server/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM golang as builder
+
+WORKDIR /usr/src/lokomotive-admission-webhook-server
+
+COPY . .
+
+RUN make MOD=vendor build-webhook
+
+# Admission webhook
+
+FROM scratch
+
+COPY --from=builder /usr/src/lokomotive-admission-webhook-server/admission-webhook-server /usr/src/admission-webhook-server/
```

`/usr/src` should contain the source code of application, not compiled binaries.

I'd use `/usr/local/bin`, as `/usr/bin` is usually used for the distro binaries.
```suggestion
COPY --from=builder /usr/src/lokomotive/admission-webhook-server /usr/local/bin/
```